<launch>
    <node name="usb_cam_node" pkg="usb_cam" type="usb_cam_node" output="screen">
        <param name="video_device" value="/dev/video2"/>
        <param name="image_width" value="640"/>
        <param name="image_height" value="480"/>
        <param name="pixel_format" value="yuyv"/>
        <param name="frame_id" value="camera"/>
    </node>

    <node name="detection_node" pkg="robot_project_pkg" type="detection.py" output="screen"/>

    <node name="rqt_image_view" pkg="rqt_image_view" type="rqt_image_view"
          args="/phone_detections/image_annotated"
          launch-prefix="gnome-terminal --"/>

    <node name="phone_detection_echo" pkg="rostopic" type="rostopic"
          args="echo /phone_detections"
          launch-prefix="gnome-terminal --"
          output="screen"/>
</launch>